ui/src/views/Tasks/TaskLog/index.jsx

```diff
@@ -21,13 +24,24 @@ import Link from '../../../utils/Link';
     right: theme.spacing.unit * 12,
   },
 }))
+@graphql(taskQuery, {
+  options: props => ({
+    fetchPolicy: 'network-only',
+    errorPolicy: 'all',
+    variables: {
+      taskId: props.match.params.taskId,
+    },
+  }),
+})
 export default class TaskLog extends Component {
   render() {
     const { classes, match, stream } = this.props;
+    const { task } = this.props.data;
     const url = decodeURIComponent(match.params.logUrl);
 
     return (
       <Dashboard title="Log" disablePadding>
+        <Helmet state={task && task.status.state} />
```

Comment:
We should grab the state from the task run. `task.graphql` could be as simple as:

```graphql
query Task($taskId: ID!) {
  task(taskId: $taskId) {
    status {
      runs {
        state
      }
    }
  }
}

```

The run number can be grabbed using `this.props.match.params.runId`.

Reply:
@helfi92 Made the changes. :)